### DISCUSSION TITLE
Can some one upload following folder for yetiforce 6.1.0

### DISCUSSION THREAD

**Discussion by rohanjhanepal** [YetiForceCompany/YetiForceCRM#14282] [General]
some how the folder resources at public_html/layouts/basic/modules/Settings/LayoutEditor/resources/
was removed , 
could some one please upload and share link of resources folder @ 
public_html/layouts/basic/modules/Settings/LayoutEditor/resources

**Comment by vovpff** [comment]
Why you can't download it directly from github?

**Reply by rskrzypczak** [reply]
https://github.com/YetiForceCompany/YetiForceCRM/tree/developer/public_html/layouts/basic/modules/Settings/LayoutEditor/resources
or
https://github.com/YetiForceCompany/YetiForceCRM/releases/tag/6.1.0
